nixos/modules/security/pam.nix

```diff
@@ -383,6 +383,24 @@ let
         '';
       };
 
+      failDelay = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = lib.mdDoc ''
+            If enabled, this will Replaces the `FAIL_DELAY' setting from login.defs.
```

Comment:
```suggestion
            If enabled, this will replace the `FAIL_DELAY' setting from <literal>login.defs</literal>.
```

Reply:
It looks like there was a little problem https://github.com/NixOS/nixpkgs/runs/7880374236?check_suite_focus=true#step:5:229.
`NotImplementedError: ('md node not supported yet', 'inline_html', ('<literal>',))`